manifests/v1beta1/installs/katib-cert-manager/kustomization.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+resources:
+  - ../katib-standalone
+  # Cert-manager certificate for webhooks
+  - certificate.yaml
+images:
+  - name: docker.io/kubeflowkatib/katib-controller
+    newName: docker.io/kubeflowkatib/katib-controller
+    newTag: latest
+  - name: docker.io/kubeflowkatib/katib-db-manager
+    newName: docker.io/kubeflowkatib/katib-db-manager
+    newTag: latest
+  - name: docker.io/kubeflowkatib/katib-ui
+    newName: docker.io/kubeflowkatib/katib-ui
+    newTag: latest
+
+patchesStrategicMerge:
```

Comment:
I think we should also create a patch to remove Katib Cert Generator RBAC: https://github.com/DavidSpek/katib/blob/fix-manifests/manifests/v1beta1/components/webhook/rbac.yaml.
ServiceAccount, ClusterRole, ClusterRoleBinding.
This RBAC is used for the `katib-cert-generator` Job.

Reply:
What about https://github.com/DavidSpek/katib/blob/fix-manifests/manifests/v1beta1/components/webhook/webhooks.yaml?

Reply:
These webhooks we should create in any install.
I believe we just use this patch: https://github.com/kubeflow/katib/blob/7054dff3fb1e29bb493c65dc6165c856a0128987/manifests/v1beta1/installs/katib-cert-manager/patches/katib-cert-injection.yaml#L6 to fill the `caBundle`.

Reply:
Cool, just wanted to be sure :).

Reply:
Just pushed a commit with these changes.

Reply:
Thanks @DavidSpek!
From my side only this comment left: https://github.com/kubeflow/katib/pull/1498#discussion_r611075289.